File: Java/Advanced Conditional Statements/Lab/12. Trade Commissions/TradeCommissions.java
import java.util.Scanner;

public class TradeCommissions {

    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        String city = input.nextLine();
        double amountSales = Double.parseDouble(input.nextLine());
        double commission = 0;
        boolean valid = true;

        switch(city){
            case "Sofia":
                if(amountSales >= 0 && amountSales <= 500){
                    commission = amountSales * 0.05;
                } else if (amountSales > 500 && amountSales <= 1000){
                    commission = amountSales * 0.07;
                }else if (amountSales > 1000 && amountSales <= 10000){
                    commission = amountSales * 0.08;
                }else if(amountSales > 10000){
                    commission = amountSales * 0.12;
                }else {
                    valid = false;
                }
                break;
            case "Plovdiv":
                if(amountSales >= 0 && amountSales <= 500){
                    commission = amountSales * 0.055;
                } else if (amountSales > 500 && amountSales <= 1000){
                    commission = amountSales * 0.08;
                }else if (amountSales > 1000 && amountSales <= 10000){
                    commission = amountSales * 0.12;
                }else if(amountSales > 10000){
                    commission = amountSales * 0.145;
                }else {
                    valid = false;
                }
                break;
            case "Varna":
                if(amountSales >= 0 && amountSales <= 500){
                    commission = amountSales * 0.045;
                } else if (amountSales > 500 && amountSales <= 1000){
                    commission = amountSales * 0.075;
                }else if (amountSales > 1000 && amountSales <= 10000){
                    commission = amountSales * 0.10;
                }else if(amountSales > 10000){
                    commission = amountSales * 0.13;
                }else {
                    valid = false;
                }
                break;
            default:
                valid = false;
        }

        if(valid){
            System.out.printf("%.2f", commission);
        }else {
            System.out.println("error");
        }
    }
}

File: Python/Basics/Conditional Statements Advanced/Lab/TradeCommissions.py
commissions = [[5,7,8,12], [4.5,7.5,10,13], [5.5,8,12,14.5]]

city = input()
sellVol = float(input())

comSize = 0
error = False

if sellVol < 0:
    print("error")
    error = True

if city == "Sofia" and not error:
    if 0 <= sellVol <= 500:
        comSize = sellVol * (commissions[0][0] / 100)
    elif 500 < sellVol <= 1000:
        comSize = sellVol * (commissions[0][1] / 100)
    elif 1000 < sellVol <= 10000:
        comSize = sellVol * (commissions[0][2] / 100)
    elif 10000 < sellVol:
        comSize = sellVol * (commissions[0][3] / 100)
elif city == "Varna" and not error:
    if 0 <= sellVol <= 500:
        comSize = sellVol * (commissions[1][0] / 100)
    elif 500 < sellVol <= 1000:
        comSize = sellVol * (commissions[1][1] / 100)
    elif 1000 < sellVol <= 10000:
        comSize = sellVol * (commissions[1][2] / 100)
    elif 10000 < sellVol:
        comSize = sellVol * (commissions[1][3] / 100)

elif city == "Plovdiv" and not error:
    if 0 <= sellVol <= 500:
        comSize = sellVol * (commissions[2][0] / 100)
    elif 500 < sellVol <= 1000:
        comSize = sellVol * (commissions[2][1] / 100)
    elif 1000 < sellVol <= 10000:
        comSize = sellVol * (commissions[2][2] / 100)
    elif 10000 < sellVol:
        comSize = sellVol * (commissions[2][3] / 100)

else:
    if not error:
        print("error")
        error = True

if not error:
    print(f"{comSize:.2f}")
